tests/test_md.py

```diff
@@ -532,6 +538,81 @@ def test_multiple_steps_local_consistency(self):
         np.testing.assert_array_equal(summed_pot_xs, xs)
         np.testing.assert_array_equal(summed_pot_boxes, boxes)
 
+    def test_multiple_steps_local_entire_system(self):
+        """Verify that running multiple_steps_local is valid even when consuming the entire system, IE radius ~= inf.
+
+        - Only a single particle, the reference, should not move
+        """
+        mol, _ = get_biphenyl()
+        ff = Forcefield.load_from_file("smirnoff_1_1_0_sc.py")
+
+        temperature = 300
+        dt = 1.5e-3
+        friction = 1.0
+        seed = 2022
+        radius = np.inf
+        num_steps = 5
+
+        unbound_potentials, sys_params, masses, coords, box = get_solvent_phase_system(
+            mol, ff, 0.0, minimize_energy=False
+        )
+        v0 = np.zeros_like(coords)
+        bps = []
+        for p, bp in zip(sys_params, unbound_potentials):
+            bps.append(bp.bind(p).bound_impl(np.float32))
+
+        # Select the molecule as the local idxs
+        local_idxs = np.arange(len(coords) - mol.GetNumAtoms(), len(coords), dtype=np.int32)
+
+        intg = LangevinIntegrator(temperature, dt, friction, masses, seed)
+
+        intg_impl = intg.impl()
+
+        ctxt = custom_ops.Context(coords, v0, box, intg_impl, bps)
+
+        xs, boxes = ctxt.multiple_steps_local(num_steps, local_idxs, radius=radius)
+
+        identical_positions = (xs == coords).sum(axis=1)
+        assert len(identical_positions) == 1, "Expected only a single atom to be stationary"
```

Comment:
This assertion seems trivially true (i.e. independent of the equality check).

Reply:
I mean that `assert len(identical_positions) == 1` is just checking that the first element of the shape is 1, but that will be true regardless of the values in `xs` and `coords`, right?

Reply:
Talking offline, this was wrong. There was a mismatch in the shapes of the two arrays anyways which made it more confusing.

Fixed in [2cb977f](https://github.com/proteneer/timemachine/pull/981/commits/2cb977fa681004b8470e5a99273e681d1305a238)